src/main/resources/init_db.sql

```diff
@@ -0,0 +1,10 @@
+CREATE DATABASE IF NOT EXISTS `jdbc`;

```

Comment:
We can replace DATABASE by SCHEMA. Why is it necessary? I don't know, but receive this comment in my work :)
```suggestion
CREATE SCHEMA IF NOT EXISTS `jdbc`;

```